### ISSUE TITLE
[accessibility] TalkBack ListView scroll not working if list is reversed

### ISSUE THREAD

**Issue by ottonascarella** [flutter/flutter#45814]
## Description

TalkBack's scroll gestures do not work with `ListView` if it's `reverse:true`.

## Steps to reproduce

```dart
import 'package:flutter/material.dart';

void main() => runApp(App());

class App extends StatelessWidget {
  final List<String> items = List.generate(77, (i) => 'Item $i');

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: Scaffold(
        appBar: AppBar(
          title: Text('Testing'),
        ),
        body: ListView.builder(
          reverse: true,
          semanticChildCount: items.length,
          itemCount: items.length,
          itemBuilder: (context, i) {
            return IndexedSemantics(
              index: i,
              child: ListTile(
                title: Text(items[i]),
              ),
            );
          },
        ),
      ),
    );
  }
}

```

if you just change code to `reverse:false`, it works just fine.

**Comment by janmoppel** [comment]
Hello @ottonascarella !

Thank you for posting an issue.
However, I tested your code on a physical device using the latest stable version, and scrolling works with both `reverse:false` and `reverse:true` options. Can you please specify, what exactly doesn't work for you and can you please post the `flutter doctor -v` command output?

Thank you.

**Comment by ottonascarella** [comment]
hm...I have not tried with physical device. Will do and get back to you!

Thanks!

**Comment by clocksmith** [comment]
@ottonascarella 

What method are you using to test talkback on a non-physical device?

**Comment by clocksmith** [comment]
Closing this for now because I cannot reproduce on a device with TalkBack. Please re-open if you are able to reproduce it.
